test/srtp_driver.c

```diff
@@ -219,12 +220,13 @@ int main(int argc, char *argv[])
     unsigned do_list_mods = 0;
     unsigned do_log_stdout = 0;
     srtp_err_status_t status;
+    size_t hdr_size = 12;
```

Comment:
Can be const?

Reply:
I thought I had tried and it failed, will double check.

Reply:
looks like it worked